tests/integration_tests/vectorstores/test_redis.py

```diff
@@ -19,8 +23,42 @@ def test_redis_new_vector() -> None:
     """Test adding a new document"""
     texts = ["foo", "bar", "baz"]
     docsearch = Redis.from_texts(
-        texts, FakeEmbeddings(), redis_url="redis://localhost:6379"
+        texts, FakeEmbeddings(), redis_url=TEST_REDIS_URL
+    )
+    docsearch.add_texts(["foo"])
+    output = docsearch.similarity_search("foo", k=2)
+    assert output == [Document(page_content="foo"), Document(page_content="foo")]
+
```

Comment:
If there's any async functionality not wrapped in a sync method you can test directly using `pytest.mark.asyncio`

Reply:
I'll add some more tests :)